jdaviz/configs/cubeviz/plugins/sonify_data/tests/test_sonify_data.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from specutils import SpectralRegion
+import astropy.units as u
+
+pytest.importorskip("strauss")
+
+
+def test_sonify_data(cubeviz_helper, spectrum1d_cube_larger):
```

Comment:
How long does this test take? Do we have to mark it as slow?

Reply:
It runs very quickly for me locally.